### DISCUSSION TITLE
no se puede editar el checkout de wp?

### DISCUSSION THREAD

**Discussion by Admintp24h** [woocommerce/woocommerce#53086] [Checkout Flow]
si ponemos el checkout de wordpress me sale bloqueado y no me deja editar ningun campo como puedo realizar cambios por ejemplo en el campo de phone si tengo en todos los bloques el candado puesto?

**Comment by nielslange** [comment]
Hello @Admintp24h 👋

Thanks for reaching out. I translated your message using Google Translate:

> If we put the WordPress checkout, it appears blocked and does not let me edit any field. How can I make changes, for example, to the phone field if I have the padlock set on all the blocks?

To assist you, please let us know what the expected result should look like. Which of the fields do you want to edit? As for the padlock on the phone filed, do you mean you want to make this field required?
